components/Gallery/Gallery.js

```diff
@@ -85,7 +85,11 @@ const Gallery = ({ items, onClose, startItemSrc, children, t }) => {
         className='pswp'
         tabIndex='-1'
         role='dialog'
-        style={{ background: '#000' }}
+        style={{
+          background: '#000',
+          top: isTester && headerHeight,
```

Comment:
Why did you change this behaviour? I think the gallery should keep going over the header – we should keep the fullscreen image experience free of headers.

Reply:
reverted